src/user/user.service.ts

```diff
@@ -60,6 +63,21 @@ export class UserService {
         return user;
     }
 
+    public async requestToBeTester(
+        testFlightRequest: TestFlightRequest
+    ): Promise<Tester> {
+        const requestTest = await this.testerRepository.findOneBy({
+            email: testFlightRequest.email.toLowerCase()
+        });
+        if (!requestTest) {
+            this.logger.verbose(
+                `New tester ${testFlightRequest.email} created`
+            );
+            return this.testerRepository.save(testFlightRequest);
```

Comment:
should log after user is created, otherwise log will show up even .save() fail

`await repo.save()
log()
`